File: app/src/main/java/com/simcom/ecashier/ui/addCollection/ErrorDialog.java
package com.simcom.ecashier.ui.addCollection;

import android.app.AlertDialog;
import android.app.Dialog;
import android.content.Context;
import android.content.DialogInterface;
import android.os.Bundle;
import android.view.View;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatDialogFragment;

import com.simcom.ecashier.R;

public class ErrorDialog extends Dialog {


    public ErrorDialog(@NonNull Context context) {
        super(context);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.dialog_error);
        findViewById(R.id.button).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                dismiss();
            }
        });
    }
}

File: app/src/main/java/com/simcom/ecashier/ui/addCollection/ErrorDialog.kt
package com.simcom.ecashier.ui.addCollection

import android.app.Dialog
import android.content.Context
import android.os.Bundle
import android.view.View
import com.simcom.ecashier.R

class ErrorDialog(context: Context) : Dialog(context) {
    override fun onCreate(savedInstanceState: Bundle) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.dialog_error)
        findViewById<View>(R.id.button).setOnClickListener { dismiss() }
    }
}